views/marts/product/dim_excludable_servers.view.lkml

```diff
@@ -13,11 +13,126 @@ view: dim_excludable_servers {
+    description: "Whether the server has reported telemetry (from front-end) only for a single day."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_single_day_telemetry_only {
+    type: yesno
+    sql: ${TABLE}.has_reason_single_day_telemetry_only ;;
+    label: "Single day (telemetry - all)"
+    description: "Whether the server has reported telemetry (both from front-end and server) only for a single day."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+
+  # Test server indications
+
+  dimension: has_reason_custom_build_version_format {
+    type: yesno
+    sql: ${TABLE}.has_reason_custom_build_version_format ;;
+    label: "Custom build version"
+    description: "Whether the server reported a custom build version format in security update check endpoint."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_ran_tests {
+    type: yesno
+    sql: ${TABLE}.has_reason_ran_tests ;;
+    label: "Ran tests"
+    description: "Whether the server reported tests ran in security update check."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_internal_email {
+    type: yesno
+    sql: ${TABLE}.has_reason_internal_email ;;
+    label: "Internal Email"
+    description: "Whether the server is registered with an internal email (i.e. `@mattermost.com`)."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_test_server {
+    type: yesno
+    sql: ${TABLE}.has_reason_test_server ;;
+    label: "Test server"
+    description: "Whether the server is in the seed file with known test servers. Data originates from seed file."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_community {
+    type: yesno
+    sql: ${TABLE}.has_reason_community ;;
+    label: "Community"
+    description: "Whether the server is the community server. Data originated from seed file."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  # Spam or invalid telemetry
+
+  dimension: has_reason_active_users__registered_users {
+    type: yesno
+    sql: ${TABLE}.has_reason_active_users__registered_users ;;
+    label: "Active users > registered users"
+    description: "Whether the server has more active users than registered users."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_no_stripe_installation_found {
+    type: yesno
+    sql: ${TABLE}.has_reason_no_stripe_installation_found ;;
+    label: "Not reported in Stripe"
+    description: "Whether the server has no installation registered in stripe."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_restricted_ip {
+    type: yesno
+    sql: ${TABLE}.has_reason_restricted_ip ;;
+    label: "Restricted IP"
+    description: "Whether the server's IP belongs to the list of restricted IP addresses."
+    view_label: "Server: Exclusion Reasons"
+  }
+
+  dimension: has_reason_invalid_server_id {
```

Comment:
1/5, I might be wrong, but I'd think that most (if not all) of these "reasons to exclude" are not "to be used" or "usable" by the Looker users for any metric/analysis, so I think that hiding them would be the right approach (unless we hear from people asking for them).

Reply:
Not sure which should be there. My idea was to initially share them, and then refine them according to most used. This way we can both make it explicit on what are the exclusion reasons, and gather feedback.

Reply:
"gather feedback", I think either way (showing them or showing only the "all") it'll be prob. difficult to gather feedback, unless we ask (have to see if Looker keeps track of the field usage count). Depending on the user proficiency - they might need or not to use the full set. My belief is that most users will want the binary good/bad server decision. The other reason that I was advocating for hiding them is that the previous models/explores were showing a lot of fields that were not really used, fwik - I didn't look to me that there was a lot of thought in deciding which ones to show/or not - making the explores prob. more complex that they needed to. 
That said, I'm ok to having them all displayed and we can prune them later (or not) depending on the feedback we receive.